src/argv.ts

```diff
@@ -50,6 +50,10 @@ export class Argv {
             writeStreams?.stderr(chalk`{black.bgYellowBright  WARN } --shell-isolation does not work with --no-shell-executor-no-image\n`);
         }
 
+        if (!argv.shellExecutor && argv.shellIsolation) {
+            writeStreams?.stderr(chalk`{black.bgYellowBright  WARN } --shell-isolation does not work with --no-shell-executor\n`);
```

Comment:
I run `gitlab-ci-local --shell-isolation` all the time, wouldn't this break my workflow?
Given that `argv.shellExecutor` defaults to false.

Reply:
Indeed, didn't think it through, removing this and the associated test